docs/deployment-and-operations/packaging.md

```diff
@@ -47,6 +47,17 @@ COPY target/statefun-example*jar /opt/statefun/modules/statefun-example/
 COPY module.yaml /opt/statefun/modules/remote/module.yaml
 {% endhighlight %}
 
+<div class="alert alert-info">
```

Comment:
While you're at this, to make the instructions more complete, I think it'll be good to also further say somewhere that we do not publish convenience images for snapshot / instable versions. For those, users should build the image from the repo (via the `./tools/docker/build-stateful-functions.sh` script).